### PR TITLE
add charset UTF-8 to template exception

### PR DESCRIPTION
**Pull request by uralbash**
before:

after:

### CODE REVIEW

**Comment by asvetlov** [comment]
I believe it's better solved by specifying correct content type for responses.
Also all rendered pages requires utf-8 perhaps.

**Comment by jettify** [comment]
hm, could we get rid from `<meta charset="UTF-8" />` in templates? Specified content type should  be enough

**Comment by uralbash** [comment]
@jettify Well, I removed it. If it is clear why this is necessary, you can always add

File: aiohttp_debugtoolbar/middlewares.py [hunk 1]
```diff
@@ -103,7 +103,9 @@ def toolbar_middleware(request):
                 #
                 # response = request.invoke_subrequest(subrequest)
                 body = tb.render_full(request).encode('utf-8', 'replace')
-                response = web.Response(body=body, status=500)
+                response = web.Response(
+                    body=body, status=500,
+                    content_type='text/html; charset=utf-8')
```
Comment (asvetlov) [inline aiohttp_debugtoolbar/middlewares.py]: The patch is wrong, sorry.

`content_type` parameter is for type only, not charset.
Ctor adds `; charset=utf-8` to given parameter ultimately: https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/web_reqrep.py#L742-L748

`content_type='text/html'` should be enough.

Perhaps we need *charset* param for `aiohttp.web.Response` ctor, I'll file the issue.

Comment (uralbash) [inline aiohttp_debugtoolbar/middlewares.py]: @asvetlov Thanks for clarification

**Pull request merged**